src/common/OutputDataSocket.cc

```diff
@@ -299,8 +299,7 @@ int OutputDataSocket::dump_data(int fd)
 {
   m_lock.Lock(); 
   list<bufferlist> l;
-  l = data;
-  data.clear();
+  l.swap(data);
```

Comment:
@tchaikov Sorry, if I confused you. I agree that the std::move is required. But I was thinking about the `data.push_back()` at L315.